openmc/deplete/results_list.py

```diff
@@ -192,3 +193,72 @@ def get_depletion_time(self):
         for ix, res in enumerate(items):
             times[ix] = res.proc_time
         return times
+
+    def export_to_materials_xml(self, burnup_index, nuc_with_data=None, new_xml_name=None):
+        """Export Materials.xml file from the depletion results
+
+        Parameters
+        ----------
+        burn_index : int
+            Index of burnup step to evaluate
+        nuc_with_data : Iterable of str, optional
+            nuclides list to evaluate with neutron data. If not provided,
```

Comment:
The first sentence could be improved because we don't really need it to be a list, just some iterable of strings. Maybe something like `Nuclides with cross section data" and then a small note on why this is necessary (some isotopes tracked through depletion do not have xs)

Reply:
Done!